test/e2e/data/infrastructure/cluster-template-kcp-remediation.yaml

```diff
@@ -0,0 +1,227 @@
+apiVersion: v1
```

Comment:
why don't we call it `cluster-template-rke2cp(or rcp)-remediation` when there is a `RKE2ControlPlane` used in the template?

Reply:
> Hey @furkatgofurov7, done. The only place where i left kcp is in tests, because i've used the standard capi `KCPRemediationSpec` test

I noticed you answered it earlier, even though we are re-using kcp spec from CAPI e2e framework, we are actually testing rke2ctpl remediation under the hood, so this make it confusing to me at the end of the day.

Reply:
@furkatgofurov7 agree, that is reasonable. I will rename it to `cluster-template-rcp-remediation.yaml`